src/sqlparser/src/ast/operator.rs

```diff
@@ -40,7 +40,7 @@ impl fmt::Display for UnaryOperator {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.write_str(match self {
             UnaryOperator::Plus => "+",
-            UnaryOperator::Minus => "-",
+            UnaryOperator::Minus => "- ", // avoid --1, - x and -x are the same.
```

Comment:
I'm assuming that `- expr` and `-expr` have the same behaviour.

Is this approach alright to avoid `- -expr` being unparsed to `--expr`?

cc @xiangjinwu

Reply:
Nevermind, I realized that these mean different things at the parsing level. `- expr` parses with `Nested`.

Reply:
Known issue. See: https://github.com/risingwavelabs/risingwave/issues/4344